utplsql-maven-plugin/src/test/resources/pom.xml

```diff
@@ -19,19 +19,46 @@
                     <goal>test</goal>
                 </goals>
                 <configuration>
-                    <suitePaths>
-                        <suitePath>suite_path1</suitePath>
-                        <suitePath>suite_path2</suitePath>
-                    </suitePaths>
-                    <sourcePaths>
-                        <sourcePath>source_path1</sourcePath>
-                        <sourcePath>source_path2</sourcePath>
-                        <sourcePath>source_path3</sourcePath>
-                    </sourcePaths>
-                    <testPaths>
-                        <testPath>test_path1</testPath>
-                        <testPath>test_path2</testPath>
-                    </testPaths>
+                
+                	<!--  Mandatory Attributes -->
+                	<url>url_of_connection</url>
```

Comment:
Maybe you can use utPLSQL-local profile and ${dbUrl}, ${dbUser}, ${dbPass} variables here as I did in https://github.com/utPLSQL/utPLSQL-java-api/blob/ab8804397366bd8fde66365305250fae68fd6e53/pom.xml?

Reply:
Same as before. Try to add a full test for the plugin.

Reply:
Changed